signin_google.feature

```diff
@@ -0,0 +1,142 @@
+Feature: 1.1 Sign in with Google account
+
+Background: 
+Given user is on the https://qa.ddso-spot.quantori.com
+And user follows "Sign in with Google"
+
+@positive
+Scenario: 1.1_01 Sign in with Google account from the list 
+Given user has a Google account
+And the sigh in form promts to choose an account 
+And user already signed in to the Google account on the device
+When user hits on his Google account 
+Then user is signed in 
+And user is on the main page 
+
```

Comment:
I love the readability of your test. I think it would be great to double-check for typo errors.

Reply:
Thank you, I will check